pkg/helmcertifier/checks/checks_test.go

```diff
@@ -97,3 +97,33 @@ func TestHasReadme(t *testing.T) {
 		})
 	}
 }
+
+func TestContainsTest(t *testing.T) {
+	type testCase struct {
+		description string
+		uri         string
+	}
+
+	positiveTestCases := []testCase{
+		{description: "tarball contains at least one test", uri: "testchart-0.1.0.tgz"},
```

Comment:
Please use normalized chart names with semantic versioning to help co-relate the artifact with the test. In the case of a valid chart, I believe using the same `chart-0.1.0-v3.valid.tgz` is fine.

Reply:
done